typescript/lib/services/service_context.ts

```diff
@@ -65,42 +66,48 @@ class ServiceContext {
     }
 
     // Docs available at https://docs.kurtosistech.com/kurtosis-libs/lib-documentation
-    public execCommand(command: string[]): [number, Uint8Array | string, Error] {
+    public async execCommand(command: string[]): Promise<[number, Uint8Array | string, Error]> {
```

Comment:
You can (and should) actually make this return a Result as well - we should use Results everywhere rather than manually using an Error type